### PR TITLE
Update Docs

### LINKED ISSUE #61: Update Lab Server & Annotation Server Readme
As soon as both Swagger API Docs are deployed and available through the VM, the ReadMe files need to be updated

### PR DESCRIPTION
**Pull request by Benjamin-Frost**
Closes #61 

> **Warning:** This PR depends on #182 as it updates the endpoint URLs

### CODE REVIEW

File: annotation-server/README.md [hunk 1]
```diff
@@ -3,8 +3,10 @@
 ## Swagger Endpoint Documentation (OpenAPI)
 
 You can find all available API endpoints defined as Swagger Documentation
-[here](http://localhost:3000/api-docs) (currently just when you run the sever on
-`localhost`, later on available through the VM).
+[here](http://172.20.24.66:8080/api).
+
+> Please note that this endpoint is only available in the HPI network.
+> That means you either need to be connected directly or through a VPN.
```
Comment (SebastianWagner2) [inline annotation-server/README.md:9]:
```suggestion
[here](http://172.20.24.66:8080/api-docs).

> Please note that this endpoint is only available in the HPI network.
> That means you either need to be connected directly or through a VPN.
```

Currently, the route to the swagger documentation is root/api-docs not root/api. We can change that though if you like.

File: lab-server/README.md [hunk 2]
```diff
@@ -3,8 +3,10 @@
 ## Swagger Endpoint Documentation (OpenAPI)
 
 You can find all available API endpoints defined as Swagger Documentation
-[here](http://localhost:3000/api-docs) (currently just when you run the sever on
-`localhost`, later on available through the VM).
+[here](http://172.20.24.66:8081/api).
+
+> Please note that this endpoint is only available in the HPI network.
+> That means you either need to be connected directly or through a VPN.
```
Comment (SebastianWagner2) [inline lab-server/README.md:9]:
```suggestion
[here](http://172.20.24.66:8081/api-docs).

> Please note that this endpoint is only available in the HPI network.
> That means you either need to be connected directly or through a VPN.
```

Same here

**Review by SebastianWagner2 — CHANGES_REQUESTED** [review]

File: annotation-server/README.md (hunk 1 above)
Comment (Benjamin-Frost) [reply annotation-server/README.md:9]: As written in the PR description this PR depends on #182 which changes exactly that (`api-docs` ➞ `api`)

File: lab-server/README.md (hunk 2 above)
Comment (Benjamin-Frost) [reply lab-server/README.md:9]: same as above

**Review by SebastianWagner2 — APPROVED** [review]

**Review by Toalaah — APPROVED** [review]

**Pull request merged**